projects/lib/src/admin/components/page-editor/page-editor.component.ts

```diff
@@ -34,6 +34,10 @@ export const EMPTY_PAGE_CONTENT_DOC: Partial<PageContentDoc> = {
 export class PageEditorComponent implements OnInit, OnChanges {
   @Input() renderSiteUrl: string;
   @Input() editorOptions: any;
+  @Input() resourceType?: // optional
+  'Products' | 'Categories' | 'Catalogs' | 'Promotions' | 'Suppliers' | 'Buyers' | 'ProductFacets' = null;
```

Comment:
Can you please include SupplierUsers and AdminUsers as well?

Reply:
@amrarick26 seems like he mirrored how the Headstart SDK currently works - maybe we want to log that in JIRA for Oliver to get fixed. For now I can push an update that allows those two on the @Input() and passes it `as any` to the SaveAssignment call.

Reply:
Hey sorry just catching up. Yeah I think its a limitation of the marketplace API that we'd want Oliver to be aware about if we need